{% extends 'base.html.twig' %}

{% block title %}Hello {{ "home" }}!{% endblock %}

{% block body %}

<style>
    .example-wrapper { margin: 1em auto; max-width: 800px; width: 95%; font: 18px/1.5 sans-serif; }
    .example-wrapper code { background: #F5F5F5; padding: 2px 6px; }
</style>

<div class="example-wrapper">
    

    This friendly message is coming from:
    <ul>
        <li>Your controller at <code><a href="{{ 'src/Controller/FrontofficeHomeController.php'|file_link(0) }}">src/Controller/FrontofficeHomeController.php</a></code></li>
        <li>Your template at <code><a href="{{ 'templates/frontoffice_home/index.html.twig'|file_link(0) }}">templates/frontoffice_home/index.html.twig</a></code></li>
    </ul>
</div>



<h1>Liste des circuits</h1>

    <table class="table">
        <thead>
<link rel="stylesheet" href="/css/bootstrap.min.css">
<script src="/js/bootstrap.min.js"></script>
            <tr>
                <th>Id</th>
                <th>...</th>
            </tr>

        </thead>
        <tbody>
	{%dump(circuits)%}
        {% for circuit in circuits %}
            <tr>
                <td>{{ circuit.id }}</td>
                <td>
		     <a href="{{ path('front_circuit_show', {id: circuit.id}) }}">d√©tails</a>
		</td>
            </tr>
        {% else %}
            <tr>
                <td colspan="2">no circuits found</td>
            </tr>
        {% endfor %}
        </tbody>


    </table>



	



{% endblock %}
